RTNeural/activation/activation_eigen.h

```diff
@@ -464,6 +464,418 @@ class PReLUActivationT
     T outs_internal alignas(RTNEURAL_DEFAULT_ALIGNMENT)[out_size];
     v_type alpha;
 };
+
+/** Dynamic implementation of an approximated GeLU activation layer. */
+template <typename T, typename MathsProvider = DefaultMathsProvider>
+class GeLUActivation : public Activation<T>
+{
+public:
+    explicit GeLUActivation(int size)
+        : Activation<T>(size, {}, "gelu")
+    {
+        inVec = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>::Zero(size, 1);
+        outVec = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>::Zero(size, 1);
+    }
+
+    GeLUActivation(std::initializer_list<int> sizes)
+        : GeLUActivation(*sizes.begin())
+    {
+    }
+
+    RTNEURAL_REALTIME inline void forward(const T* input, T* out) noexcept override
+    {
+        inVec = Eigen::Map<const Eigen::Matrix<T, Eigen::Dynamic, 1>, RTNeuralEigenAlignment>(
+            input, Layer<T>::in_size, 1);
+
+        static const T sqrt_2_over_pi = T(0.45015815807);
+        outVec = inVec.array().unaryExpr([sqrt_2_over_pi](T x) {
+            T xCube = x * x * x;
+            return 0.5 * x * (1.0 + MathsProvider::tanh(sqrt_2_over_pi * (x + 0.044715 * xCube)));
```

Comment:
```suggestion
            return (T) 0.5 * x * ((T) 1 + MathsProvider::tanh(sqrt_2_over_pi * (x + (T) 0.044715 * xCube)));
```
We should probably give `0.044715` a name as well.